app/policies/post_policy.rb

```diff
@@ -0,0 +1,25 @@
+class PostPolicy < ApplicationPolicy
+  def index?
+    true
+  end
```

Comment:
I tried to implement what you had but took myself to remove the link from unauthorized users.  Is this good practice? 

https://github.com/jgrant29/bloccit/tree/authorization <https://github.com/jgrant29/bloccit/tree/authorization>

Justin 

> On Jul 1, 2015, at 1:04 PM, Bobbilee <notifications@github.com> wrote:
> 
> In app/policies/post_policy.rb <https://github.com/jgrant29/bloccit/pull/1#discussion_r33712691>:
> 
> > @@ -0,0 +1,25 @@
> > +class PostPolicy < ApplicationPolicy
> > +  def index?
> > +    true
> > +  end
> well u have it as true so all users should be able to see the index of posts.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/jgrant29/bloccit/pull/1/files#r33712691>.
>